### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by alexey-zolotarev**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -4,11 +4,16 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# How to use an unbound check box column to select grid rows
+# Data Grid for Windows Forms - How to use an unbound check box column to select grid rows
 
+The [Grid View](https://docs.devexpress.com/WindowsForms/3464/controls-and-libraries/data-grid/views/grid-view) in the [Data Grid](https://docs.devexpress.com/WindowsForms/3455/controls-and-libraries/data-grid) control supports a built-in checkbox column for multiple row selection.
```
Comment (albertov05) [inline Readme.md:9]: On line 7 above you mention "check box", but on this line you say "checkbox". Can you please make it consistent?

File: Readme.md [hunk 2]
```diff
@@ -4,11 +4,16 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# How to use an unbound check box column to select grid rows
+# Data Grid for Windows Forms - How to use an unbound check box column to select grid rows
 
+The [Grid View](https://docs.devexpress.com/WindowsForms/3464/controls-and-libraries/data-grid/views/grid-view) in the [Data Grid](https://docs.devexpress.com/WindowsForms/3455/controls-and-libraries/data-grid) control supports a built-in checkbox column for multiple row selection. 
+Set the following properties as follows to enable this checkbox column:
```
Comment (albertov05) [inline Readme.md]:
```suggestion
Set the properties below as follows to enable this checkbox column:
```

File: Readme.md [hunk 3]
```diff
@@ -4,11 +4,16 @@
...
+In v18.1 and later versions, you can sync row selected states with a Boolean field in the bound data source. A checkbox column's bound mode ensures that row selection persists whenever you filter or group grid data. See the following topic for more information: [Multiple Row Selection via Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding)
```
Comment (albertov05) [inline Readme.md]:
```suggestion
In v18.1 and later versions, you can sync row selected states with a Boolean field in the bound data source. A checkbox column's bound mode ensures that row selection persists whenever you filter or group grid data. See the following topic for more information: [Multiple Row Selection Using Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding).
```

Comment (albertov05) [reply Readme.md]: Can you please update the actual topic title as well?

File: Readme.md [hunk 4]
```diff
@@ -4,11 +4,16 @@
...
 
-Starting with version **18.1**, in multiple row selection mode, you can sync row selected states with a Boolean field in the bound data source. The selection binding mode ensures that row selection persists whenever you filter or group grid data. Refer to the [Multiple Row Selection via Built-In Check Column and Selection Binding](https://documentation.devexpress.com/WindowsForms/16439/Controls-and-Libraries/Data-Grid/Focus-and-Selection-Handling/Multiple-Row-Selection-via-Built-In-Check-Column-and-Selection-Binding) help article for more information.
-
-## ***If the built-in CheckBox column does not suit you, we will be happy to find the most appropriate solution for you if you describe your real-life scenario. Just click   [here](https://www.devexpress.com/Support/Center/Question/Create) to submit a ticket to our Support Center.***
+## Documentation
+- [Multiple Row Selection via Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding)
```
Comment (albertov05) [inline Readme.md]:
```suggestion
- [Multiple Row Selection Using Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding)
```

File: Readme.md [hunk 5]
```diff
@@ -4,11 +4,16 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# How to use an unbound check box column to select grid rows
+# Data Grid for Windows Forms - How to use an unbound check box column to select grid rows
```
Comment (alexey-zolotarev) [inline Readme.md]:
```suggestion
# Data Grid for Windows Forms - How to use an unbound checkbox column to select grid rows
```

**Pull request merged**